solidity/dashboard/src/css/liquidity-page.less

```diff
@@ -43,16 +77,71 @@
       display: flex;
       flex-direction: column;
       justify-content: center;
+      position: relative;
+
+      .liquidity__info-tile__tooltip {
+        width: 12px;
+        height: 12px;
+        position: absolute;
+        top: 0px;
+        right: 8px;
+        cursor: pointer;
+
+        .tooltip__content-wrapper {
+          text-align: left;
+          min-width: 16rem;
+        }
+      }
+
+      h6 {
+        * {
+          text-transform: none;
```

Comment:
Do we need this? If so why? 🤔

Reply:
Look like I forgot to delete it before the commit, doesn't look like it's used anywhere so i deleted it.